frontend/src/services/gqlAuth.js

```diff
@@ -7,6 +8,16 @@ module.exports = {
     return !isTokenExpired(token);
   },
 
+  getRole() {
+    var token = localStorage.getItem('jwt_user_token');
```

Comment:
When we log in we generate a token with data from floods_private, and we keep the role in there. This prevents the need to query the private table when checking permissions on functions.

[decode()](https://github.com/auth0/jwt-decode) allows us to get the json from the token.

I haven't done a deep dive into jwt related security concerns, but from a front end stand point it seems as though react-router has more security issues (for example: "authorized: true" would open up a route) than the jwt itself.

The most important thing in my mind from a security stand point is the backend. If someone can get to a protected route it probably isn't the end of the world, but if an unauthorized user can modify data then we're getting into trouble.